src/content/docs/en/guides/authentication.mdx

```diff
@@ -188,6 +188,78 @@ const session = await getSession(Astro.request);
 - [`auth-astro` on GitHub](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#auth-astro)
 - [Auth.js documentation](https://authjs.dev/)
 
+## Clerk
+
+Clerk is a complete suite of embeddable UIs, flexible APIs, and admin dashboards to authenticate and manage your users. An [official Clerk SDK for Astro](https://clerk.com/docs/references/astro/overview) is available.
+
+### Installation
+
+Install `@clerk/astro` using the package manager of your choice.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install @clerk/astro
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add @clerk/astro
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add @clerk/astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuration
+
+Follow [Clerk's own Astro Quickstart guide](https://clerk.com/docs/quickstarts/astro) to set up Clerk integration and middleware in your Astro project.
+
+### Usage
+
+Clerk provides components that allow you to control the visibility of pages based on your user's authentication state. Show logged out users sign in button instead of content available to users who are logged in:
```

Comment:
```suggestion
Clerk provides components that allow you to control the visibility of pages based on your user's authentication state. Show logged out users a sign in button instead of the content available to users who are logged in:
```
I think the articles help the sentence here! So assuming we'd be on the same page, I'll just commit and get this published so we can start the weekend!  ;)